scripts/build-win.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const tp = require('teen_process');
+const path = require('path');
+
+const APKS_DIR = path.resolve('.', 'apks');
+const APK_FILENAME = 'ApiDemos-debug.apk';
+const APK_ORIGIN_PATH = path.resolve('.', 'app', 'build', 'outputs', 'apk', 'debug', 'app-debug.apk');
+const APK_DEST_PATH = path.join('apks', APK_FILENAME);
+
+async function runGradleBuild() {
+  try {
+    await tp.exec('cmd.exe', ['/c', 'gradlew.bat', 'clean', 'assembleDebug']);
+  } catch (e) {
+    throw new Error(`Cannot perform 'clean assembleDebug' Gradle tasks: ${e.stderr || e.message}`);
+  }
+}
+
+async function copyAPK() {
+  try {
+    await fs.promises.access(APKS_DIR, fs.constants.R_OK);
+  } catch (e) {
+    await fs.promises.mkdir(APKS_DIR, { recursive: true });
+  }
+  await fs.promises.copyFile(APK_ORIGIN_PATH, APK_DEST_PATH);
```

Comment:
copy is usually slow. Consider replacing it with https://nodejs.org/docs/latest/api/fs.html#fspromisesrenameoldpath-newpath